nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -96,6 +96,44 @@
   The options are named identically for all other display managers.
   </para>
  </simplesect>
+ <simplesect xml:id="sec-x11--graphics-cards-intel">
+  <title>Intel graphics drivers</title>
+  <para>
+   There are two choices for Intel graphics driver in X.org:
```

Comment:
```suggestion
   There are two choices for Intel graphic drivers in X.org:
```

Reply:
This is actually correct, but should capitalized: [Intel Graphics](https://en.wikipedia.org/wiki/Intel_HD_and_Iris_Graphics) is a trademark.